### PR TITLE
Add MDC properties for  BillableUsage messages

### PR DESCRIPTION
**Pull request by barnabycourt**
This is an experimental add to set OrgId and account_id when processing BillableUsage events in swatch-producer-aws. This will create a properties block on the event in splunk that contains each of the MDC attributes. After verification of how well this works we can examine whether it is worth spreading the pattern more widely or reverting.

### CODE REVIEW

File: swatch-producer-aws/src/main/java/com/redhat/swatch/processors/BillableUsageProcessor.java [hunk 1]
```diff
@@ -78,13 +79,18 @@ public BillableUsageProcessor(
...
     if (billableUsage == null) {
       log.warn("Skipping null billable usage: deserialization failure?");
       return;
     }
+    if (billableUsage.getOrgId() != null) {
+      MDC.put("OrgId", billableUsage.getOrgId());
+    }
+    if (billableUsage.getAccountNumber() != null) {
+      MDC.put("account_id", billableUsage.getAccountNumber());
+    }
```
Comment (kahowell) [inline swatch-producer-aws/src/main/java/com/redhat/swatch/processors/BillableUsageProcessor.java:92]: nitpick: can we please be consistent with casing (I prefer snake_casing personally)? So, OrgId -> org_id

File: swatch-producer-aws/src/main/java/com/redhat/swatch/processors/BillableUsageProcessor.java [hunk 2]
```diff
@@ -78,13 +79,18 @@ public BillableUsageProcessor(
...
-    if (log.isDebugEnabled()) {
-      log.debug("Picked up billable usage message {} to process", billableUsage);
-    }
+    log.debug("Picked up billable usage message {} to process", billableUsage);
     if (billableUsage == null) {
       log.warn("Skipping null billable usage: deserialization failure?");
       return;
     }
+    if (billableUsage.getOrgId() != null) {
+      MDC.put("OrgId", billableUsage.getOrgId());
```
Comment (awood) [inline swatch-producer-aws/src/main/java/com/redhat/swatch/processors/BillableUsageProcessor.java:88]:
```suggestion
      MDC.put("org_id", billableUsage.getOrgId());
```

**Comment by lindseyburnett** [comment]
Confirmed that properties appear in splunk

image

**Review by lindseyburnett — APPROVED** [review]

**Pull request merged**